src/validation.cpp

```diff
@@ -1233,8 +1232,8 @@ CAmount GetMasternodePayment(int nHeight, CAmount blockValue, bool fMNRewardReal
     }
 
     // TODO remove this when we re-organize testnet
-    if (Params().NetworkIDString() == CBaseChainParams::TESTNET) fMNRewardReallocated = false;
-    if (fMNRewardReallocated) {
+    // if (Params().NetworkIDString() == CBaseChainParams::TESTNET) fV20Active = false;
```

Comment:
@UdjinM6 @PastaPastaPasta Same here

Reply:
we need to keep them if we want to be able to sync on testnet before #5619

Reply:
also, it's much cleaner to drop them in #5619 and not here even if we want to merge them one after another imo